test/sampletests/withverify/verifySampleFailures.js

```diff
@@ -1,14 +1,17 @@
 module.exports = {
   before(client) {
     client.globals.calls++;
+    client.url('http://localhost');
   },
 
   demoTest(client) {
-    client.url('http://localhost')
-      .verify.equal(0, 1, 'custom message')
-      .verify.elementPresent('#badElement')
+    client.verify.equal(0, 1, 'custom message')
       .verify.elementPresent('#weblogin')
-      .end();
+      .verify.elementPresent('#badElement');
```

Comment:
why do we need to remove the `end()` ?

Reply:
I removed while adding one more test in **verifySampleFailures.js**. I added back now in after hook.

Reply:
looks good